libs/content/workspace/src/task_manager.rs

```diff
@@ -649,7 +649,7 @@ impl TaskManager {
     #[instrument(level = "warn", skip(self), fields(thread = format!("{:?}", thread::current().id())))]
     fn background_load(&self, request: LoadRequest) {
         let id = request.id;
-        let content_result = self.core.read_document_with_hmac(id);
+        let content_result = self.core.read_document_with_hmac(id, true);
```

Comment:
I think this could also be triggered by a sync pulling a change to this file

Reply:
thanks for having a look, i think that will still lead to good data (compared to what is in it's absence)